src/core/interfaces.ts

```diff
@@ -88,11 +88,22 @@ export type Bounds = {
 };
 
 /**
+ * Client bounds
+ */
+export interface IEntityBounds {
```

Comment:
This is replacing `SvgElement.getBBox()` with a compatible call implemented by the canvas drawer. Sadly `Bounds` has a different interface that doesn't match. We can't change the DOM's API so I think we still need this.